src/main/java/org/apache/accumulo/access/AeNode.java

```diff
@@ -200,7 +203,7 @@ private AndNode(List<AeNode> children) {
 
     @Override
     char operator() {
-      return '&';
+      return AND_OPERATOR;
```

Comment:
This method returns a `char`, so I think we want to leave it unless from where it's called its being cast to a `byte`.